### PR TITLE
Add Capture tutorials

### PR DESCRIPTION
**Pull request by torbsorb**
Same as C++. 

Note that 
```csharp
var camera = zivid.ConnectCamera(new Zivid.NET.Settings("camera_settings/set1/frame_01.yml"));
```
is intentionally left out, as that code is buggy.

### CODE REVIEW

File: source/Camera/Basic/CaptureTutorial.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+## Introduction
+
+This tutorial describes how to use Zivid SDK to capture point clouds and 2D images.
+
+1. [Initialize](#initialize)
+2. [Connect](#connect)
+   1. [Specific Camera](#connect---specific-camera)
+   2. [Virtual Camera](#connect---virtual-camera)
+3. [Configure](#configure)
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I vote for this structure:

3. [Configure](#configure)
   1. [Capture Assistant](#capture-assistant)
   2. [Manual Configuration](#manual-configuration)
      1. [Single](#single-frame)
      2. [HDR](#hdr-frame)
      3. [2D](#2d-settings)
   3. [From File](#from-file)

File: source/Camera/Basic/CaptureTutorial.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+   1. [Capture Assistant](#capture-assistant)
+   2. [Manual Configuration](#manual-configuration)
+      1. [Single](#single-frame)
+      2. [HDR](#hdr-frame)
+      3. [From File](#from-file)
+      4. [2D](#2d-settings)
+4. [Capture](#capture)
+    1. [HDR](#capture-hdr)
+    2. [2D](#capture-2d)
+5. [Save](#save)
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Can we add:

5. [Save](#save)
    1. [2D](#save-2d)

Like we did in python: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598

File: source/Camera/Basic/CaptureTutorial.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+foreach (var iris in irisList)
+{
+    Console.WriteLine("Add settings for frame with iris = " + iris);
+    var settings = new Zivid.NET.Settings();
+    settings.Iris = iris;
+    settingsList.Add(settings);
+}
+```
+
+### From File
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: If comment from line 3 is accepted, this should go after line 153.

File: source/Camera/Basic/CaptureTutorial.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+Zivid Studio can store the current settings to .yml files. These can be read and applied in the API. You may find it easier to modify the settings in these (human-readable) yaml-files in your preferred editor.
+```csharp
+camera.SetSettings(new Zivid.NET.Settings("frame_01.yml"));
+```
+
+### 2D Settings
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: If comment from line 3 is accepted, this should go before line 146. Also one extra # is needed.

File: source/Camera/Basic/CaptureTutorial.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+    2. 60Hz is found in (amongst others) Japan, Americas, Taiwan, South Korea and Philippines.
+    3. 50Hz is found in most rest of the world.
+
+### Manual configuration
+
+We may choose to configure settings manually ([go to source][settings-url]). For more information about what each settings does, please see [Zivid One+ Camera Settings][kb-camera_settings-url].
+
+#### Single Frame
+
+We can configure settings for an individual frame directly to the camera.
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Move ([go to source][settings-url]) from line 105 here, like in: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598

File: source/Camera/Basic/CaptureTutorial.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+```csharp
+camera.UpdateSettings(s =>
+{
+    s.Iris = 10;
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Put iris to 20, like in: 4406342f5eae91e649fa393a8bb1bc9f253ae598
We will make a separate PR to change the source samples

File: source/Camera/Basic/CaptureTutorial.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+    s.BlueBalance = 1.081;
+    s.RedBalance = 1.709;
+});
+```
+
+#### HDR Frame
+
+We may also set a list of settings to be used in an [HDR capture](#capture-hdr).
+```csharp
+var irisList = new List<ulong>() { 20, 25, 30 };
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Change iris values to 14, 21, 35 like in: 4406342f5eae91e649fa393a8bb1bc9f253ae598
There is already a PR to change these values for python sample. We will make on for other languages as well.

File: source/Camera/Basic/CaptureTutorial.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+    settings.Iris = iris;
+    settingsList.Add(settings);
+}
+```
+
+### From File
+
+Zivid Studio can store the current settings to .yml files. These can be read and applied in the API. You may find it easier to modify the settings in these (human-readable) yaml-files in your preferred editor.
+```csharp
+camera.SetSettings(new Zivid.NET.Settings("frame_01.yml"));
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Is the path not needed here? I guess the reason is that we don't have a source sample yet?

File: source/Camera/Basic/CaptureTutorial.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+```
+
+### 2D Settings
+
+It is possible to only capture a 2D image. This is faster than a 3D capture, and can be used . 2D settings are configured as follows ([go to source][settings2d-url]).
+```csharp
+var settings = new Zivid.NET.Settings2D()
+{
+    Iris = 22,
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Change iris to 35, like in https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598
We will make another PR to change source samples and make then consistent between languages

File: source/Camera/Basic/CaptureTutorial.md [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+```csharp
+var hdrFrame{ Zivid.NET.HDR.Capture(camera, settingsList) };
+```
+It is possible to [manually create](#hdr-frame) the `settingsList`, if not set via [Capture Assistant](#capture-assistant).
+
+### Capture 2D
+
+If we only want to capture a 2D image, which is faster than 3D, we can do so via the 2D API ([go to source][capture2d-url]).
+```csharp
+var frame = camera.Capture2D(settings);
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Should we call this frame2D? I assume it is not the same as a 3D frame, right? If it is not, it should have a different name to avoid confusion.

For python it is called frame_2d: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598

File: source/Camera/Basic/CaptureTutorial.md [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+
+## Save
+
+We can now save our results ([go to source][save-url]).
+```csharp
+var resultFile = "result.zdf";
+frame.Save(resultFile);
+```
+The API detects which format to use. See [Point Cloud][kb-point_cloud-url] for a list of supported formats.
+
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Add section for Save 2D, like in: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598

File: source/Camera/Basic/QuickCaptureTutorial.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+```csharp
+var frame = camera.Capture();
+```
+
+## Save
+
+We can now save our results. By default the 3D point cloud is saved in Zivid format `.zdf` ([go to source][save-url]).
+```csharp
+var resultFile = "result.zdf";
+frame.Save(resultFile);
```
Comment (SatjaSivcev) [inline source/Camera/Basic/QuickCaptureTutorial.md]: Simplify to one line for this quick tutorial?

frame.Save("result.zdf");

Like in: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598

File: source/Camera/Basic/CaptureTutorial.md [hunk 13]
```diff
@@ -0,0 +1,214 @@
...
+Zivid Studio can store the current settings to .yml files. These can be read and applied in the API. You may find it easier to modify the settings in these (human-readable) yaml-files in your preferred editor.
+```csharp
+camera.SetSettings(new Zivid.NET.Settings("frame_01.yml"));
+```
+
+### 2D Settings
+
+It is possible to only capture a 2D image. This is faster than a 3D capture, and can be used . 2D settings are configured as follows ([go to source][settings2d-url]).
+```csharp
+var settings = new Zivid.NET.Settings2D()
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Should we call this settings2D? If it is different than settings for 3D, then we should.

For python we do call them differently, see: https://github.com/zivid/python-samples/pull/55/commits/4406342f5eae91e649fa393a8bb1bc9f253ae598

File: source/Camera/Basic/CaptureTutorial.md (hunk 10 above)
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: See line 157 comment. If accepted, change settigns variable name.

File: source/Camera/Basic/CaptureTutorial.md (hunk 8 above)
Comment (torbsorb) [reply source/Camera/Basic/CaptureTutorial.md]: https://zividlabs.atlassian.net/browse/CS-401

File: source/Camera/Basic/QuickCaptureTutorial.md (hunk 12 above)
Comment (SatjaSivcev) [reply source/Camera/Basic/QuickCaptureTutorial.md]: You forgot this one. I pushed a commit for that now.
